source/_components/fan.xiaomi_miio.markdown

```diff
@@ -13,7 +13,7 @@ ha_iot_class: "Local Polling"
 ha_release: 0.57
 ---
 
-The `xiaomi_miio` fan platform allows you to control the Xiaomi Air Purifier and Xiaomi Air Humidifier.
+The `xiaomi_miio` fan platform allows you to control the Xiaomi Mi Air Purifier 2, Air Purifier 2S, Air Purifier Pro, Air Humidifier and Air Fresh.
```

Comment:
Would it make sense to make a generic description and then create a list like in https://www.home-assistant.io/components/switch.tplink/ -- that would make it easier to read (IMO). Looks good otherwise!

Reply:
Good point! I will update the PR this evening.